src/main/java/it/pagopa/pm/gateway/controller/PostePayPaymentTransactionsController.java

```diff
@@ -439,6 +438,10 @@ private PostePayPollingResponse createPollingResponse(String requestId, PaymentR
         response.setRequestId(entity.getGuid());
         response.setCorrelationId(entity.getCorrelationId());
         response.setIsOnboarding(entity.getIsOnboarding());
+        if (ObjectUtils.isNotEmpty(entity.getStatus())){
+            PaymentRequestStatusEnum paymentRequestStatusEnum = PaymentRequestStatusEnum.of(entity.getStatus());
+            response.setPaymentRequestStatus(paymentRequestStatusEnum);
```

Comment:
This can maybe be rewritten on a single line, since paymentRequestStatusEnum is not used anymore.
Also, method PaymentRequestStatusEnum.of() throws an IllegalArgumentException which is never caught by the caller, so it's better if you catch it somewhere to make your code stronger

Reply:
what do you mean with "since paymentRequestStatusEnum is not used anymore." ?
I'm adding a catch for the exception